pkg/controller/workloaddispatcher/dispatcher/controllers.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dispatcher
+
+import (
+	ctrl "sigs.k8s.io/controller-runtime"
+
+	"sigs.k8s.io/kueue/pkg/controller/workloaddispatcher/incrementaldispatcher"
+	"sigs.k8s.io/kueue/pkg/util/admissioncheck"
+)
+
+func SetupControllers(mgr ctrl.Manager, namespace string, dispatcherName string) error {
```

Comment:
```suggestion
func SetupControllers(mgr ctrl.Manager, namespace string, dispatcherName string) (string, error) {
```

Let us return the failed dispatcher name.